iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/balancer/RegionBalancer.java

```diff
@@ -147,6 +151,7 @@ private LoadManager getLoadManager() {
 
   public enum RegionGroupAllocatePolicy {
     COPY_SET,
```

Comment:
Maybe we can also remove this incomplete copyset implementation, which will never be used

Reply:
Fixed